src/components/todo/index.js

```diff
@@ -1,3 +1,5 @@
+import './todo.css';
+
 export default function Todo({ name, content, id, removeTodo }) {
   const handleRemove = () => {
     removeTodo(id);
```

Comment:
As mention above, I think it will be great if you organize your code in the suggested way!